### DISCUSSION TITLE
How to debug the HTTP request block?

### DISCUSSION THREAD

**Discussion by nauvalazhar** [AutomaApp/automa#725] [General]
Hi, is it possible to debug the HTTP request block, and how?

Thank you!

**Comment by Kholid060** [comment]
It's currently not possible using Automa, but you can try using postman or [hoppscotch.io](https://hoppscotch.io/) to do it

**Reply by nauvalazhar** [reply]
Okay, so I need to test the request first until it works, and implement it in Automa?
